### PR TITLE
Update draft branch

### PR DESCRIPTION
**Pull request by zolkis**
***
<a href="https://pr-preview.s3.amazonaws.com/zolkis/wot-scripting-api/pull/217.html" title="Last updated on Jun 4, 2020, 3:13 PM UTC (4afd851)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/wot-scripting-api/217/2297459...zolkis:4afd851.html" title="Last updated on Jun 4, 2020, 3:13 PM UTC (4afd851)">Diff</a>

### CODE REVIEW

**Comment by zolkis** [comment]
@danielpeintner PTAL
This is just for updating the 'draft' branch which I intend to use on Discovery API experimentation without disturbing the master.

**Comment by zolkis** [comment]
We should also do a branch cleanup on the w3c repo. 
Only `master` and `draft` should remain, if possible.

File: index.html [hunk 1]
```diff
@@ -1093,183 +1966,203 @@ <h2>ExposedThing Examples</h2>
...
+    </p>
 
-    <section> <h3>The <dfn>discover()</dfn> method</h3>
+    <section>
+      <h3>The <dfn>start()</dfn> method</h3>
       <p>
-        Starts the discovery process that will provide <a>ThingDescription object</a>s of <a>Thing Description</a>s that match an optional <var>filter</var> argument which can specify the discovery method and match a specific source URL, a query and a template object. Check the  <a href="#discovery-examples">examples</a>. The method MUST run the following steps:
+=======
```
Comment (danielpeintner) [inline index.html:2026]: This needs to be removed

**Comment by danielpeintner** [comment]
You merged `master` into `draft`, correct?

There is at least one issue in Section "9.1" marked with the diff marker "======= ".

Can you point me to the changes that are still in draft? If there are none left I would suggest creating a *fresh* and clean branch.
It is very difficult in the current way to identify possible issues....

File: index.html [hunk 2]
```diff
@@ -1093,183 +1966,203 @@ <h2>ExposedThing Examples</h2>
...
+        The <dfn>fragment</dfn> property represents a template object used for matching property by property against discovered <a>Thing</a>s.
       </p>
+    </section>
 
-      <section> <h4>The <dfn>start()</dfn> method</h4>
-      <p>
+    <section>
+      <h3>The <dfn>start()</dfn> method</h3>
+      <div>
+>>>>>>> 3df8c825b88b73c5a313909980a4497d46eda5cb
```
Comment (danielpeintner) [inline index.html:2109]: another merge issue

File: index.html (hunk 1 above)
Comment (zolkis) [reply index.html:2026]: Looks like the merge went wrong and I missed this one.

**Comment by zolkis** [comment]
> I would suggest creating a fresh and clean branch

Yes, merging master to draft.
We could remove the draft branch and then re-create it but we'd lose all history on the draft branch.

**Comment by zolkis** [comment]
OK, I copied the master over the draft. But if you want, I can remove the draft branch and create a new one.

**Comment by zolkis** [comment]
Merging it for now, then will remove all branches except master.
We will re-create the `draft` branch as needed.

**Pull request merged**